include/openthread-types.h

```diff
@@ -485,6 +506,9 @@ typedef enum otMeshcopTlvType
 /**
  * This structure represents an MLE Link Mode configuration.
  */
+#ifdef _WIN32
```

Comment:
Replace with something like `OT_TOOL_ALIGN()` and place definition in `include/platform/toolchain.h`?